### ISSUE TITLE
Xamarin

### ISSUE THREAD

**Issue by swift11** [ZeBobo5/Vlc.DotNet#340]
Hello,

any plans to support Xamarin.Forms ?

**Comment by jeremyVignelles** [comment]
yes, there are, but I'm not a Xamarin developer. Do you want to contribute?

**Comment by ahmad-crossplatform** [comment]
I would like to contribute with it. how to do so ?

**Comment by jeremyVignelles** [comment]
Great, we can discuss on gitter (not now because I'm leaving). We are also discusssing with @mfkl to make the future Videolan-backed .net library. We also need help here : https://github.com/mfkl/LibVLCSharp . He is also better than me for Xamarin stuffs, and we have designed the architecture for that project.

**Comment by mfkl** [comment]
Thanks for the shout out @jeremyVignelles 

Added a README with current status. As indicated, Android integration is blocked by a mono runtime bug. iOS needs some build script modifications to bundle and expose C headers as it's currently not the case. I'm looking into it now. Check out up for grabs issues if you wanna help.

**Comment by ahmad-crossplatform** [comment]
Hello , well regarding Android I have started a Project to port the Java code to C# of the libvlc Android  so maybe we can work on that and somehow we make one big solution . I Think it would be better than depending on the vlcandroid Community giving us the latest aar files, plus i have had really hard time to compile the vlc Android ...
I have just created a gitter account and I have a slack account . 

https://github.com/ahmad-crossplatform/Libvlc.Xamarin.Android

**Comment by jeremyVignelles** [comment]
Now that libvlcsharp is on the tracks, I think that we can close this issue : It is out of scope with this project's goal.

See https://github.com/videolan/libvlcsharp for a Xamarin libvlc library.
